Update filenames and README (#5)
slack-samples/deno-welcome-bot: README.md

executed. A trigger file describes a scenario in which a workflow should be run,
such as a user pressing a button or when a specific event occurs.

### `/datastores`

> [Datastores](https://api.slack.com/future/datastores) are used to store and
> retrieve data related to your application. Required scopes to use datastores
> include `datastores:write` and `datastores:read`.

Reviewer: I copied some of the language used on the docs so that it's slightly more clear that datastores are an optional feature, but feel free to revert or otherwise modify this! And regardless of if copy changes are reverted, I think we should try to land the changes to the required scopes before merging!

```suggestion
[Datastores](https://api.slack.com/future/datastores) can securely store and
retrieve data in your application. Required scopes to use datastores include
`datastore:write` and `datastore:read`.
```

Author: Love it, save for the "in". You're retrieving data _from_ the datastore -- it's outside of your application. Right?

Reviewer: Ah yes!!
